modules/core-updates/8.1.md

```diff
@@ -0,0 +1,92 @@
+* Customer Settings > Customer page is now multistore compatible [PR#27608](https://github.com/PrestaShop/PrestaShop/pull/27608)
+* Introduced `PrestaShop\PrestaShop\Core\Security\PasswordGenerator` class to generate random passwords with multiples types/lengths [PR#31004](https://github.com/PrestaShop/PrestaShop/pull/31004)
+* Introduced `PrestaShop\PrestaShop\Core\Security\Hashing` class to hash a password from a password and a salt with md5 [PR#31004](https://github.com/PrestaShop/PrestaShop/pull/31004)
+* Introduced `actionProductPriceCalculation` hook [PR#27927](https://github.com/PrestaShop/PrestaShop/pull/27927)
+* Added a new Smarty variable `theme_dir` in front controllers [PR#30383](https://github.com/PrestaShop/PrestaShop/pull/30383)
+* Added a feature to disable `core.js` loading on custom themes [PR#29995](https://github.com/PrestaShop/PrestaShop/pull/29995). [More informations][corejs-informations]
+* Added supplier url and manufacturer url to Smarty `{url}` helper [PR#30242](https://github.com/PrestaShop/PrestaShop/pull/30342) and [PR#30314](https://github.com/PrestaShop/PrestaShop/pull/30314)
+
+## Deprecations
+
+### PHP
+
+* `PrestaShop\PrestaShop\Core\String\CharacterCleaner` → Its use is not required
+* `Customer::validateController()` → The password check has been moved in controllers and this method is not called anywhere since 1.7.0
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\BooleanColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\BooleanColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\ColorColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ColorColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\Common\IdentifierColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\IdentifierColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\DataColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\DataColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\DisableableLinkColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\DisableableLinkColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\LinkGroupColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\LinkGroupColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\OrderPriceColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Order\OrderPriceColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\PreviewColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\PreviewColumn`
+* `Prestashop\PrestaShop\Core\Grid\Column\Type\PrivateColumn` → moved to `PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\PrivateColumn`
+* `CmsController::$cms` → use now `CmsController::getCms()`
+* `CmsController::$cms_category` → use now `CmsController::getCmsCategory()`
+* `Validate::isAnything()` → will be removed from 9.0 since returns always `true`
+
+## Updated dependencies
+
+### PHP libraries
```

Comment:
not sure if we should describe it since they are adds and not updates, but since we bring api-platform/core and league/oauth2-server, it may be interesting to keep it on this page